### DISCUSSION TITLE
Prevent floating windows from launching behind sketchybar

### DISCUSSION THREAD

**Discussion by szfkamil** [FelixKratz/SketchyBar#519] [General]
I have been experiencing this recurring issue of floating windows launching behind SketchyBar for a while, as shown in the screenshot. I was not able to find a solution for a few weeks.

![Screenshot 2024-03-04 at 15 10 06](https://github.com/FelixKratz/SketchyBar/assets/82778513/f56af5a0-f24c-4d4c-92d8-1b4559e759ac)

**Comment by FelixKratz** [comment]
Try `sketchybar --bar topmost=off` this will make floating windows spawn on top of the bar.

**Reply by szfkamil** [reply]
Yup, this was it!
